miio/airqualitymonitor_miot.py

```diff
@@ -0,0 +1,286 @@
+
+    @property
+    def humidity(self) -> int:
+        """Return humidity value (0...100%)."""
+        return self.data["humidity"]
+
+    @property
+    def pm25(self) -> int:
+        """Return PM 2.5 value (0...1000ppm)."""
+        return self.data["pm25"]
+
+    @property
+    def pm10(self) -> int:
+        """Return PM 10 value (0...1000ppm)."""
+        return self.data["pm10"]
+
+    @property
+    def temperature(self) -> float:
+        """Return temperature value (-30...100°C)."""
+        return self.data["temperature"]
+
+    @property
+    def co2(self) -> int:
+        """Return co2 value (0...9999ppm)."""
+        return self.data["co2"]
+
+    @property
+    def battery(self) -> int:
+        """Return battery level (0...100%)."""
+        return self.data["battery"]
+
+    @property
+    def charging_state(self) -> ChargingState:
+        """Return charging state."""
+        return ChargingState(self.data["charging_state"])
+
+    @property
+    def monitoring_frequency(self) -> int:
+        """Return monitoring frequency time (0..600 s)."""
+        return self.data["monitoring_frequency"]
+
+    @property
+    def screen_off(self) -> int:
+        """Return screen off time (0..300 s)."""
+        return self.data["screen_off"]
+
+    @property
+    def device_off(self) -> int:
+        """Return device off time (0..60 min)."""
+        return self.data["device_off"]
+
+    @property
+    def display_temperature_unit(self):
+        """Return display temperature unit."""
+        return DisplayTemeratureUnitCGDN1(self.data["temperature_unit"])
+
+    def __repr__(self) -> str:
+        s = (
+            "<AirQualityMonitorCGDN1Status humidity=%s, "
+            "pm25=%s, "
+            "pm10=%s, "
+            "temperature=%s, "
+            "co2=%s, "
+            "battery=%s, "
+            "charging_state=%s"
+            "monitoring_frequency=%s"
+            "screen_off=%s"
+            "device_off=%s"
+            "display_temerature_unit=%s>"
+            % (
+                self.humidity,
+                self.pm25,
+                self.pm10,
+                self.temperature,
+                self.co2,
+                self.battery,
+                self.charging_state,
+                self.monitoring_frequency,
+                self.screen_off,
+                self.device_off,
+                self.display_temperature_unit,
+            )
+        )
+        return s
+
+
+class AirQualityMonitorCGDN1(MiotDevice):
+    """Qingping Air Monitor Lite."""
+
+    def __init__(
+        self,
+        ip: str = None,
+        token: str = None,
+        start_id: int = 0,
+        debug: int = 0,
+        lazy_discover: bool = True,
+    ) -> None:
+        super().__init__(_MAPPING_CGDN1, ip, token, start_id, debug, lazy_discover)
+        self._monitoring_frequency_options = MonitoringFrequencyCGDN1
+        self._screen_off_options = ScreenOffCGDN1
+        self._device_off_options = DeviceOffCGDN1
```

Comment:
Do you have a specific use case for these (underscored variables are generally considered internal in python, i.e., not to be used anyone outside the class itself)?

Reply:
No, I just saw how it's done in `vacuum` case, but here it should be exposed to be used publicly

Reply:
Okay, I think this can be merged now. Just one more very nitpicky issue, I'm a bit unsure how useful those are, as you could access them anyway simply be importing the corresponding classes? I'm not against having them there, but I don't personally see anyone using `self.screen_off_options.SomeOption` instead of `ScreenOffCGDN1.SomeOption`.

What do you think?

Reply:
I thought there is a kind of convention to having them here, but yeah, I don't any advantage keeping in here either. Maybe one of the reasons would be to explicitly say, those enums are somehow bound with this particular class, but they can be accessed directly anyway. So I'll remove them